tasks/determine-image-expiration-task.yaml

```diff
@@ -56,7 +56,7 @@ spec:
       echo "Target branch: ${PIPELINE_TARGET_BRANCH}"
```

Comment:
(not changed here, but)
I just realized that I'd recommend using `metadata.annotations['pipelinesascode.tekton.dev/branch']` for `PIPELINE_TARGET_BRANCH` instead of the current `metadata.annotations['build.appstudio.redhat.com/target_branch']`. I feel `pipelinesascode.tekton.dev` things will be more stable than `build.appstudio.redhat.com` ones.